src/main/java/net/snowflake/ingest/connection/JWTManager.java

```diff
@@ -22,7 +22,7 @@
  *
  * @author obabarinsa
  */
-final class JWTManager extends SecurityManager {
+public class JWTManager extends SecurityManager {
```

Comment:
lets also keep this class final

Reply:
Remove final since `mockito.spy()` is used to simulate the case encounter invalid token. There might a [solution](https://stackoverflow.com/questions/14292863/how-to-mock-a-final-class-with-mockito) which required updating some dependencies. Do you think we should keep final and do this?

Reply:
ahh I see, missed the nuances of final and mockito. 

I do see powermock in our dependency, can you try using that? 

This is a security class and I would like to avoid anyone even remotely trying to extend this class. Let me know if you have any issues, we can chat offline.

Reply:
I've achieved it through PowerMockito, thanks for the tips!

Reply:
Good stuff! Thank you!